projects/stretch_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -95,25 +95,25 @@ AGENT:
         - robot_start_compass
         - joint
       allowed_actions:
-        - base_velocity
-        - rearrange_stop
-        # - move_forward
-        # - turn_left
-        # - turn_right
-        # - stop
+        # - base_velocity
+        # - rearrange_stop
+        - stop
+        - move_forward
+        - turn_left
+        - turn_right
       arm_joint_mask: [0, 0, 0, 0, 0, 0, 0] # the arm joints that the policy can control
       max_displacement: 0.25                # used when training the policy; could be different from the eval values
       max_turn_degrees: 30.0
       min_turn_degrees: 5.0
       min_displacement: 0.1
       sensor_height: 160
       sensor_width: 120
-      terminate_condition: continuous_stop
+      terminate_condition: discrete_stop #continuous_stop
       nav_goal_seg_channels: 2
 
     NAV_TO_REC:
-      type: heuristic # heuristic (default) or rl
-      checkpoint_path: data/checkpoints/ovmm/nav_to_rec_may26_teleport_dist_0_1_77%.pth
+      type: rl # heuristic (default) or rl
```

Comment:
Should we just create separate config files for RL vs Heuristic eval and pass them in on the command line?